R/reportRenovation.R

```diff
@@ -25,130 +37,151 @@ reportRenovation <- function(gdx, brickSets = NULL, silent = TRUE) {
 
 
 
+  # FUNCTION -------------------------------------------------------------------
+
+  # Replace the pattern if it is a separate vector entry
+  .replaceInVec <- function(pattern, replacement, char) {
+    sub(paste0("^", pattern, "$"), replacement, char)
+  }
+
+  # Replace the pattern if it is surrounded by "{}"
+  .replaceInVariableName <- function(pattern, replacement, char) {
+    sub(paste0("\\{", pattern, "\\}"), paste0("\\{", replacement, "\\}"), char)
```

Comment:
You don't need to escape replacement strings and with fixed you can also avoid this in the pattern:
```suggestion
    sub(paste0("{", pattern, "}"), paste0("{", replacement, "}"), char, fixed = TRUE)
```

Reply:
True, thanks!